packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -221,6 +235,7 @@ const NewNetwork = decorate([
               encapsulation,
               encrypted,
               mtu: mtu !== '' ? +mtu : undefined,
+              preferredCenterId: center !== undefined ? center.id : undefined,
```

Comment:
`createPrivateNetwork` can resolve the preferred center ID, example https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/xo/index.js#L683.

Reply:
What if the `preferredCenterId` is undefined ?

Reply:
It's correctly handled by `resolveId`.

Reply:
In your example: host is the host object or id?

Reply:
It can be an object or ID.